### PR TITLE
Fixtures redux

### PR DESCRIPTION
**Pull request by commuterjoy**
@tom-parker 

After being a bit embarrassed by the state of the fixtures as I was explaining it to you I've refactored it a little bit.

### CODE REVIEW

File: tests/server/filters/isValidSourceSpec.js [hunk 1]
```diff
@@ -2,29 +2,23 @@
...
 
-var rawSqs = JSON.parse(fs.readFileSync('./tests/server/fixtures/ingest', { encoding: 'utf8' }));
-var rawSqs__valid_message = JSON.parse(fs.readFileSync('./tests/server/fixtures/ingest--valid-message', { encoding: 'utf8' }));
-
 describe('Source validation', function () {
 	
 	it('Mark messages without action, category, source as invalid', done => {
-		var e = new EventModel(rawSqs);
-		isValidSource(e).then(event => {
+		isValidSource(fixtures.get('event')).then(event => {
```
Comment (commuterjoy) [inline tests/server/filters/isValidSourceSpec.js]: Basically removed all the fixture code and replaced with `fixtures.get('event')`

**Comment by tom-parker** [comment]
Nice :+1:

File: tests/server/filters/isValidSourceSpec.js [hunk 2]
```diff
@@ -2,32 +2,34 @@
...
 	});
 	
 	it('Must specify the action, category, source of an event', done => {
-		var e = new EventModel(rawSqs__valid_message);
-		isValidSource(e).then(event => {
-			expect(event.isValid).to.be.true;
-		});
-		done();
+		let e = fixtures.sqs.get('valid-event');
+		sinon.stub(e, 'pluck', () => 'meep');
```
Comment (commuterjoy) [inline tests/server/filters/isValidSourceSpec.js]: then we can start to stub the parts of the model

**Comment by commuterjoy** [comment]
covered elsewhere

**Pull request closed without merging**